docs/authentication/enterprise-connections/oidc/custom-provider.mdx

```diff
@@ -29,33 +29,48 @@ To make the setup process easier, it's recommended to keep two browser tabs open
   1. Add the **Name** of the connection.
   1. Add the **Key** of the provider. This is the provider's unique identifier (cannot be changed after creation).
   1. Add the **Specific Domain** that you want to allow this connection for. This is the domain of the users you want to allow to sign in to your app.
   1. Select **Add connection**. You will be redirected to the connection's configuration page.
 
-  The next steps involve configuring both your IdP and Clerk. Keep both dashboards open as you'll need to reference them.
-
   ### Configure your IdP
 
   1. If necessary, create a new app in your IdP.
-  1. In the connection's configuration page in the Clerk Dashboard, copy the **Authorized redirect URI**.
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
+  1. Copy the **Authorized redirect URI**.
   1. Add the value to your IdP's whitelisted URLs.
 
   ### Set the Discovery Endpoint, Client ID, and Client Secret in Clerk
 
-  1. In your IdP settings, copy your application's **Discovery Endpoint**, **Client ID**, and **Client Secret**.
-  1. In the connection's configuration page in the Clerk Dashboard, paste these values in their respective fields.
-  1. Add the minimum required scopes based on the IdP's documentation. Common OIDC scopes include `openid`, `email`, and `profile`.
+  Most IdPs provide a **Discovery Endpoint** to retrieve metadata about an OIDC provider. If your IdP doesn't offer this endpoint or if you need greater control over the setup process, select **Use Manual Configuration** in the **Identity Provider Configuration** section to manually configure the connection.
+
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
+  1. Copy your application's **Discovery Endpoint**, **Client ID**, and **Client Secret**.
+  1. In the connection's settings page in the Clerk Dashboard, under **Identity Provider Configuration**, paste the values in their respective fields.
+  1. Under **Scopes**, add the minimum required scopes based on the IdP's documentation if needed. Common OIDC scopes include `openid`, `email`, and `profile`.
   1. Select **Save**.
 
   ### Configure attribute mapping (optional)
 
-  If your provider returns claims in a non-standard format, use the **Attribute Mapping** section on the connection's configuration page to adjust the mapping of Clerk's user properties to match the IdP's claim attributes. Important Clerk user properties include `email`, `firstName`, and `lastName`.
+  Attribute mapping allows you to map the IdP's claims with Clerk's user properties such as the `email_verified`. OIDC Enterprise connections require the [`email_verified` claim](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims:~:text=Section%C2%A05.7.-,email_verified,-boolean) to verify email ownership. However, some IdPs, such as Microsoft Azure Active Directory, might not return this claim or use a non-standard format.
```

Comment:
```suggestion
  Attribute mapping allows you to map the IdP's claims with Clerk's user properties. OIDC enterprise connections require the [`email_verified` claim](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims:~:text=Section%C2%A05.7.-,email_verified,-boolean) to verify email ownership. However, some IdPs, such as Microsoft Azure Active Directory, might not return this claim or use a non-standard format.
```

Reply:
Changed this section to be more similar to it's previous form https://github.com/clerk/clerk-docs/pull/1769/commits/07e1f1a0f8db3b9c07d2bee928f3d460e8da362c

It's not required for most users so I think keeping it brief is better to avoid confusion 🙂